### PR TITLE
git-blame: add Korean translation

### PR DESCRIPTION
**Pull request by ellijs**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]
Remember that for translations you can send more than 1 translated page in 1 PR, just keep 1 commit per file and I recommend no more than 15 pages. This doesn't apply to the main «English» pages.

File: pages.ko/common/git-blame.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# git blame
+
+> 각각의 파일 라인에 커밋 해쉬와 마지막 작성자를 보여줍니다.
+> 더 많은 정보: <https://git-scm.com/docs/git-blame>.
+
+- 각각의 라인에 작성자의 이름과 커밋 해쉬가 포함된 파일 출력:
+
+`git blame {{file}}`
```
Comment (navarroaxel) [inline pages.ko/common/git-blame.md:8]: Please translate the placeholders, like `file`, for get this easier for non-English speakers.

Comment (ellijs) [reply pages.ko/common/git-blame.md]: Addressed and updated 👍 Thank you! @navarroaxel

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**